src/psyclone/transformations.py

```diff
@@ -2913,16 +2787,14 @@ def apply(self, node, options=None):
 
 
 # For Sphinx AutoAPI documentation generation
-__all__ = ["KernelTrans",
-           "OMPLoopTrans",
+__all__ = ["OMPLoopTrans",
```

Comment:
Please could you rearrange this list so that it's in alpha order.

Reply:
Done, I think